client/app/vps/cloud-database/index.js

```diff
@@ -0,0 +1,32 @@
+import component from './vps-cloud-database.component';
+
+import vpsCloudDatabaseOrderModule from './order';
+
+const moduleName = 'vpsCloudDatabaseModule';
+
+angular
+  .module(moduleName, [
+    vpsCloudDatabaseOrderModule,
+  ])
+  .component(component.name, component)
+  .config(/* @ngInject */($stateProvider) => {
+    $stateProvider
+      .state('iaas.vps.detail.cloud-database', {
+        resolve: {
+          shouldNotBeCA: /* @ngInject */ (
```

Comment:
Maybe not yours but would it make sense to avoid negation and have a `shouldBeCA` and return the opposite ?

Reply:
Hmm `shouldBeCa` is incorrect cause we have to reject the transition if user is in CA. Or at least rename it as `rejectIfCA` but it seems to be the same as the original name, don't you think so ?

Reply:
@marie-j Ok forget my previous comment, this will change since we will create a service who returns available feature, so the resolve will change